couchbase/json-buckets.yml

```diff
@@ -0,0 +1,1084 @@
+type: json
+data:
+  url: /pools/default/buckets/
+  server: ${COUCHBASE_HOSTPORT}
+  basicHttpAuthUsername: ${COUCHBASE_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${COUCHBASE_BASICAUTH_PASSWORD}
+
+observation:
+  - name: buckets
+    metricNamespace: couchbase
+    path: $.[?(@.name=${bucketName})]
+
+    metric:
+      - name: bg.wait.time.avg
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.avg_bg_wait_time[0]
+        type: double_gauge
+        label: average background wait
+        unit: sec
+        description: Average background wait time
+
+      - name: disk.commit.time.avg
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.avg_disk_commit_time[0]
+        type: double_gauge
+        label: average commit time
+        unit: sec
+        description: Average disk commit time
+
+      - name: disk.update.time.avg
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.avg_disk_update_time[0]
+        type: double_gauge
+        label: average update time
+        unit: microsec
+        description: Average disk update time
+
+      - name: bg.wait.total
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.bg_wait_total[0]
+        type: double_gauge
+        label: bytes read
+        unit: bytes
+        description: Average background wait time
+
+      - name: bytes.read
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.bytes_read[0]
+        type: double_gauge
+        label: bytes read
+        unit: bytes
+        description: Bytes read
+
+      - name: bytes.written
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.bytes_written[0]
+        type: double_gauge
```

Comment:
If this is really a gauge it will be completely useless. Same goes for other metrics defined as gauge here.

Reply:
there is no counter metrics, i have metrics data for nearly all metrics, check and confirm no counter.